src/Vex/Texture.h

```diff
@@ -211,9 +215,15 @@ struct TextureSubresource
     u16 mipCount = GTextureAllMips;
     u32 startSlice = 0;
     u32 sliceCount = GTextureAllSlices;
+    // Refers to the slice in DX12 and the aspect mask in Vulkan
+    TextureAspect::Flags aspect = TextureAspect::Color;
```

Comment:
This could one day end up being confusing since I believe Vk also has the concept of planes, it's probably fine, since I currently don't see any features which would require us exposing planes.

Reply:
According to their docs, that is not necessarily the case.

<img width="1106" height="176" alt="image" src="https://github.com/user-attachments/assets/0ab4f65c-0a16-4ddb-8e3f-5f6109800d9c" />
From this page: https://vulkan.lunarg.com/doc/view/1.4.309.0/mac/antora/spec/latest/chapters/formats.html#_identification_of_formats